.github/workflows/sam_package_code.yml

```diff
@@ -99,3 +99,33 @@ jobs:
           path: |
             .aws-sam/build
             README.md
+
+      - name: Set ECR Repository
+        run: echo "ECR_REPOSITORY=fhir-facade-repo" >> "$GITHUB_ENV"
+
+      - name: Determine Image Tag
+        id: determine-image-tag
+        run: |
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+            echo "IMAGE_TAG=PR-${{ github.event.pull_request.number }}" >> "$GITHUB_ENV"
+          elif [[ "${{ github.ref }}" == "refs/heads/main" ]]; then
+            VERSION=$(cat VERSION)
```

Comment:
I dont think this will work getting the version number
Its probably better to change this so this file has a new input of version number and commit id, change pull_request.yml and release.yml to pass these through and use those rather than have an if statetment